.github/workflows/rpm-build-and-test.yml

```
@@ -552,10 +576,12 @@ jobs:
     # https://github.com/actions/runner/issues/491#issuecomment-926924523
     if: |
       needs.Import-commit-pragmas.outputs.run-gha == 'true' &&
+      needs.Calc-functional-hardware-matrix.outputs.matrix != '[]' &&
       (!cancelled()) &&
       (needs.Build-RPM.result == 'success' ||
        needs.Build-RPM.result == 'skipped') &&
-      needs.Functional.result == 'success'
+      (needs.Functional.result == 'success' ||
+      needs.Functional.result == 'skipped')
```

Minor. Probably not blocking.
```suggestion
       needs.Functional.result == 'skipped')
```